x/concentrated-liquidity/pool_test.go

```diff
@@ -7,23 +7,7 @@ import (
 	"github.com/osmosis-labs/osmosis/v13/x/concentrated-liquidity/types"
 )
 
-func (s *KeeperTestSuite) TestCalcOutAmtGivenIn() {
-	currPrice := sdk.NewDec(5000)
-	currSqrtPrice, err := currPrice.ApproxSqrt() // 70.710678118654752440
-	s.Require().NoError(err)
-	currTick := math.PriceToTick(currPrice) // 85176
-	lowerPrice := sdk.NewDec(4545)
-	s.Require().NoError(err)
-	lowerTick := math.PriceToTick(lowerPrice) // 84222
-	upperPrice := sdk.NewDec(5500)
-	s.Require().NoError(err)
-	upperTick := math.PriceToTick(upperPrice) // 86129
-
-	defaultAmt0 := sdk.NewInt(1000000)
-	defaultAmt1 := sdk.NewInt(5000000000)
-
-	swapFee := sdk.ZeroDec()
-
+func (s *KeeperTestSuite) TestCalcAndSwapOutAmtGivenIn() {
```

Comment:
Note to reviewer: Significantly cleaned up pool tests by merging Calc and Swap tests into one. I know we generally don't want to do this, but this makes complete sense for two reasons:

1. These functions work hand in hand. Calc is run first and should be non mutative. If it mutates, we will see in the following Swap test.
2. Prior to this merge, these two tests used the exact same test cases and was a lot of unnecessary boilerplate